pkg/kubelet/eviction/eviction_manager_test.go

```diff
@@ -785,6 +923,233 @@ func makeContainersByQOS(class v1.PodQOSClass) []v1.Container {
 	}
 }
 
+func TestPIDPressure(t *testing.T) {
```

Comment:
Could you look at disk pressure tests and follow a similar idea. I tried to combine some of the constants and give them more useful names.

Reply:
thanks this looks good!